src/components/materialization/create/index.tsx

```diff
@@ -283,6 +282,14 @@ function MaterializationCreate() {
                     },
                 };
 
+                Object.keys(resourceConfig).forEach((collectionName) => {
+                    draftSpec.bindings[collectionName] = {
+                        resource: {
+                            ...resourceConfig[collectionName].data,
+                        },
+                    };
+                });
```

Comment:
I'll move to a reusable function soon. Look through the store of configs and generate the bindings.